packages/terra-application-links/src/tabs/ApplicationTabs.scss

```diff
@@ -0,0 +1,169 @@
+      align-items: center;
+      display: inline-flex;
+      max-width: 100%;
+    }
+
+    span {
+      flex: 0 1 auto;
+      min-width: 0;
+    }
+
+    svg {
+      flex: 0 0 auto;
+      font-size: 0.571rem;
+      margin-left: var(--terra-application-tabs-menu-arrow-left-margin, 0.5rem);
+    }
+  }
+
+  .tab,
+  .tab-menu {
+    background-image: var(--terra-application-tabs-background-image, linear-gradient(to bottom, #007ee0, #007ee0));
+    background-position: var(--terra-application-tabs-background-position, top-left);
+    background-repeat: no-repeat;
+    background-size: var(--terra-application-tabs-background-size, 0 4px);
+    border-right: var(--terra-application-tabs-border-right, 1px solid #dedfe0);
+    color: var(--terra-application-tabs-color, #fff);
+    cursor: pointer;
+    flex: 0 0 auto;
+    font-size: var(--terra-application-tabs-font-size, 1rem);
+    height: 100%;
+    line-height: var(--terra-application-tabs-line-height, 1.4);
+    max-width: 100%;
+    padding-left: var(--terra-application-tabs-padding-horizontal, 1.071rem);
+    padding-right: var(--terra-application-tabs-padding-horizontal, 1.071rem);
+    position: relative;
+    text-decoration: none;
+    transition-duration: var(--terra-application-tabs-transition-duration, 0.35s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-application-tabs-transition-timing, ease);
+    width: var(--terra-application-tab-width, 16.714rem);
+
+    &:focus {
+      border-radius: 0;
+      color: var(--terra-application-tabs-focus-color, #fff);
+    }
+
+    &:hover {
+      background-color: var(--terra-application-tabs-hover-background-color, rgba(0, 0, 0, 0.25));
+      color: var(--terra-application-tabs-hover-color, #fff);
+    }
+
+    &[aria-current='true'] {
+      background-color: var(--terra-application-tabs-active-background-color, #fff);
+      background-size: var(--terra-application-tabs-active-background-size, 100% 4px);
+      color: var(--terra-application-tabs-active-color, #000);
+      font-weight: 600;
+
+      &:hover {
+        background-image: var(--terra-application-tabs-hover-background-image, linear-gradient(to bottom, #c8cacb, #c8cacb));
+        background-size: var(--terra-application-tabs-hover-background-size, 100% 4px);
+      }
+    }
+
+    &:first-child {
+      border-left: var(--terra-application-tabs-border-left, 1px solid #dedfe0);
+      border-width: var(--terra-application-tabs-border-width-first, 0 1px 0 1px);
+    }
+  }
+
+  .tab-menu.menu-hidden {
+    display: none;
+  }
+
+  .hidden-tab {
+    color: var(--terra-application-tabs-hidden-color, #1c1f21);
+    display: block;
+    padding: var(--terra-application-tabs-hidden-selected-padding, 1.214rem 1.071rem);
+    text-align: left;
+    text-decoration: none;
+    text-overflow: unset;
+    transition-duration: 0;
+    white-space: normal;
+    word-break: break-word;
+    word-wrap: break-word;
+    
+    &:focus,
```

Comment:
Do the hidden tabs need hover, visited, & aria=current styling?

Reply:
Never mind, I see this is meant for the hidden tabs that are presented in the menu dropdown